env.local.php

```diff
@@ -1,47 +0,0 @@
-<?php
```

Comment:
We have corrected both of the points you mentioned. Is there anything else? Or shall we raise a new pull request?